integreat_cms/cms/views/utils/user.py

```diff
@@ -0,0 +1,27 @@
+from django.db.models import Q
+from django.utils.translation import gettext_lazy as _
+
+from integreat_cms.cms.models.users.user import User
+
+from ...models import Page
+
+
+def get_pages_observer_has_access_to(user: User) -> list:
+    """
+    This function checks which pages an user of a region has access to.
+
+    :param user: The user for which the access rights are checked
+    """
+    pages_user_has_access_to = Page.objects.filter(Q(authors=user) | Q(editors=user))
```

Comment:
```suggestion
    pages_user_has_access_to = Page.objects.prefetch_related('authors', 'editors').filter(Q(authors=user) | Q(editors=user))
```
I'm frustrated to learn that there seems to be no good way to annotate whether the user is in the `authors` relation and the same for `editors`, but at least we can prefetch those so that we only have one query total for the function

Reply:
Good call!